README.md

```diff
@@ -1,707 +1,1799 @@
 # Reselect
 
+[![npm package][npm-badge]][npm][![Coveralls][coveralls-badge]][coveralls][![GitHub Workflow Status][build-badge]][build]![TypeScript][typescript-badge]
+
 A library for creating memoized "selector" functions. Commonly used with Redux, but usable with any plain JS immutable data as well.
 
-- Selectors can compute derived data, allowing Redux to store the minimal possible state.
-- Selectors are efficient. A selector is not recomputed unless one of its arguments changes.
-- Selectors are composable. They can be used as input to other selectors.
+- Selectors can compute derived data, allowing `Redux` to store the minimal possible state.
+- Selectors are efficient. A selector is not recomputed unless one of its arguments changes.
+- Selectors are composable. They can be used as input to other selectors.
+
+The **Redux docs usage page on [Deriving Data with Selectors](https://redux.js.org/usage/deriving-data-selectors)** covers the purpose and motivation for selectors, why memoized selectors are useful, typical `Reselect` usage patterns, and using selectors with `React-Redux`.
+
+## Installation
+
+### Redux Toolkit
+
+While `Reselect` is not exclusive to `Redux`, it is already included by default in [the official Redux Toolkit package](https://redux-toolkit.js.org) - no further installation needed.
+
+```ts
+import { createSelector } from '@reduxjs/toolkit'
+```
+
+### Standalone
+
+For standalone usage, install the `Reselect` package:
+
+#### Using `npm`
+
+```bash
+npm install reselect
+```
+
+#### Using `yarn`
+
+```bash
+yarn add reselect
+```
+
+#### Using `bun`
+
+```bash
+bun add reselect
+```
+
+#### Using `pnpm`
+
+```bash
+pnpm add reselect
+```
+
+---
+
+## Basic Usage
+
+`Reselect` exports a [`createSelector`] API, which generates memoized selector functions. [`createSelector`] accepts one or more [`input selectors`], which extract values from arguments, and a [`combiner`] function that receives the extracted values and should return a derived value. If the generated [`output selector`] is called multiple times, the output will only be recalculated when the extracted values have changed.
```

Comment:
:hand: 

- There's a bunch of inline code names that also have brackets.  Are these supposed to be links?
- Typically inline code formatting is used for actual variable / function names.  "combiner" and "input selector" are descriptive terms and shouldn't be formatted as code.

Reply:
- Yes the brackets are links.
- I will fix the inline formatting.

Thanks for pointing these out!